flink-end-to-end-tests/flink-end-to-end-tests-pulsar/src/test/java/org/apache/flink/tests/util/pulsar/PulsarSourceOrderedE2ECase.java

```diff
@@ -29,11 +29,15 @@
 import org.apache.flink.tests.util.pulsar.cases.FailoverSubscriptionContext;
 import org.apache.flink.tests.util.pulsar.common.FlinkContainerWithPulsarEnvironment;
 import org.apache.flink.tests.util.pulsar.common.PulsarContainerTestEnvironment;
+import org.apache.flink.testutils.junit.FailsOnJava11;
+
+import org.junit.experimental.categories.Category;
 
 /**
  * Pulsar E2E test based on connector testing framework. It's used for Failover & Exclusive
  * subscription.
  */
+@Category(value = {FailsOnJava11.class})
```

Comment:
Why not just add a dependency on jaxb-api in the Java 11 profile?...

Actually, why is this necessary at all if we copy the jaxb-api jar in the pom?

Reply:
Is there a ticket for that?

Reply:
Does the connector as a whole not work on Java 11, or is it an issue with the pulsar test classes?